### PR TITLE
0.36: Turn Contract ACTION sidecars ON in testnet + mainnet

### PR DESCRIPTION
**Pull request by david-bakin-sl**
Turn Contract ACTION sidecars ON in testnet + mainnet

Signed-off-by: David Bakin <117694041+david-bakin-sl@users.noreply.github.com>

### CODE REVIEW

**Review by Nana-EC — APPROVED** [review]

File: hedera-node/hedera-app/src/test/resources/bootstrap.properties [hunk 1]
```diff
@@ -90,7 +90,7 @@ expiry.throttleResource=expiry-throttle.json
 contracts.redirectTokenCalls=true
 contracts.referenceSlotLifetime=31536000
 contracts.scheduleThrottleMaxGasLimit=5000000
-contracts.sidecars=CONTRACT_STATE_CHANGE,CONTRACT_BYTECODE
+contracts.sidecars=CONTRACT_STATE_CHANGE,CONTRACT_BYTECODE,CONTRACT_ACTION
```
Comment (Neeharika-Sompalli) [inline hedera-node/hedera-app/src/test/resources/bootstrap.properties:93]: I think we should add to `hedera-services/hedera-node/hedera-mono-service/src/main/resources/bootstrap.properties` for default values

Comment (Neeharika-Sompalli) [reply hedera-node/hedera-app/src/test/resources/bootstrap.properties:93]: oh nvmd , its correct in that file

**Review by Neeharika-Sompalli — APPROVED** [review]

**Review by lukelee-sl — DISMISSED** [review]
LG

**Review by tinker-michaelj — APPROVED** [review]
LGTM

**Review by lukelee-sl — APPROVED** [review]

**Review by Nana-EC — APPROVED** [review]

**Review by Neeharika-Sompalli — APPROVED** [review]

**Review by tinker-michaelj — APPROVED** [review]
LGTM

**Pull request merged**